src/R/Editor/Application.Test/Formatting/AutoFormatTest.cs

```diff
@@ -17,13 +17,17 @@ namespace Microsoft.R.Editor.Application.Test.Formatting {
     public class AutoFormatTest : IDisposable {
         private readonly IExportProvider _exportProvider;
         private readonly EditorHostMethodFixture _editorHost;
+        private readonly bool _autoFormat;
 
         public AutoFormatTest(REditorApplicationMefCatalogFixture catalogFixture, EditorHostMethodFixture editorHost) {
             _exportProvider = catalogFixture.CreateExportProvider();
             _editorHost = editorHost;
+            _autoFormat = REditorSettings.AutoFormat;
```

Comment:
Sure, all tests for the code that rely on `REditorSettings` static properties should be non-parallel.